test/JDBC/expected/parallel_query/with_recompile-vu-verify.out

```diff
@@ -0,0 +1,2991 @@
+  Query Text: select * from tb_recomp_71 where c71 = 0
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Index Scan using tb_recomp_71_pkey on tb_recomp_71
+              Index Cond: (c71 = 0)
+  Query Text: select set_config('plan_cache_mode', 'force_generic_plan', false)
+  ->  Result
+  Query Text: select * from tb_recomp_71 where c71 = "@p"
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Bitmap Heap Scan on tb_recomp_71
+              Recheck Cond: (c71 = 0)
+              ->  Bitmap Index Scan on tb_recomp_71_pkey
+                    Index Cond: (c71 = 0)
+  Query Text: select * from tb_recomp_71 where c71 = 0
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Index Scan using tb_recomp_71_pkey on tb_recomp_71
+              Index Cond: (c71 = 0)
+~~END~~
+
+set babelfish_showplan_all off
+go
+SELECT current_setting('plan_cache_mode') as plan_mode_after_call
+go
+~~START~~
+text
+auto
+~~END~~
+
+
+select set_config('plan_cache_mode', 'auto', false)
+go
+~~START~~
+text
+auto
+~~END~~
+
+set babelfish_showplan_all on
+go
+exec    p_recomp_71 0 with recompile    -- sets plan_cache_mode=force_generic_plan
+go
+~~START~~
+text
+Query Text: EXEC    p_recomp_71 0 /* EXECUTE WITH RECOMPILE */
+  Query Text: select * from tb_recomp_71 where c71 = "@p"
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Bitmap Heap Scan on tb_recomp_71
+              Recheck Cond: (c71 = 0)
+              ->  Bitmap Index Scan on tb_recomp_71_pkey
+                    Index Cond: (c71 = 0)
+  Query Text: select * from tb_recomp_71 where c71 = 0
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Index Scan using tb_recomp_71_pkey on tb_recomp_71
+              Index Cond: (c71 = 0)
+  Query Text: select set_config('plan_cache_mode', 'force_generic_plan', false)
+  ->  Result
+  Query Text: select * from tb_recomp_71 where c71 = "@p"
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Bitmap Heap Scan on tb_recomp_71
+              Recheck Cond: (c71 = 0)
+              ->  Bitmap Index Scan on tb_recomp_71_pkey
+                    Index Cond: (c71 = 0)
+  Query Text: select * from tb_recomp_71 where c71 = 0
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Index Scan using tb_recomp_71_pkey on tb_recomp_71
+              Index Cond: (c71 = 0)
+~~END~~
+
+set babelfish_showplan_all off
+go
+SELECT current_setting('plan_cache_mode') as plan_mode_after_call
+go
+~~START~~
+text
+auto
+~~END~~
+
+
+
+--==== end of tests ======
+go
+select set_config('plan_cache_mode', 'auto', false)
+go
+~~START~~
+text
+auto
+~~END~~
+
+SELECT current_setting('plan_cache_mode') as plan_mode_after
+go
+~~START~~
+text
+auto
+~~END~~
+
+SELECT current_setting('enable_indexscan') as enable_indexscan_after
+go
+~~START~~
+text
+off
+~~END~~
```

Comment:
The default value is `on`, hence we should set it to `on`.

Reply:
done